package.json

```diff
@@ -29,12 +29,13 @@
     "build:prod": "cross-env NODE_ENV=production yarn _build",
     "build:node": "yarn _babel && yarn build:ts",
     "build:ts": "tsc -p ./tsconfig.json",
+    "build:test": "yarn tsc -p ./test/tsconfig.json",
```

Comment:
also, either way, you don't need the `yarn` here

Reply:
good point! I'm just used to doing that to ensure the local compiler.

I didn't want the tests to have the same config as the main one. Also currently have a limitation in that I needed the test files to generate JS in place. I couldn't figure out how to have the relative imports work otherwise.